corehq/ex-submodules/casexml/apps/phone/management/commands/sync_log_debugger.py

```diff
@@ -51,3 +60,13 @@ def handle(self, *args, **options):
                             log_names[j],
                             ', '.join(case_diff)
                         )
+
+        if options['debugger']:
+            union_of_ids = set().union(*[set(log.get_footprint_of_cases_on_phone()) for log in logs])
+            intersection_of_ids = set().intersection(*[set(log.get_footprint_of_cases_on_phone()) for log in logs])
+            import pdb
```

Comment:
`ipdb` is a lifechanger if you aren't already using it: https://pypi.python.org/pypi/ipdb